### ISSUE TITLE
Vbo memory leak

### ISSUE THREAD

**Issue by nillerusr** [ptitSeb/gl4es#238]
glMapBuffer returns not real vbo. As a result game freezes and i have memory leak with vbo. Im not sure, but...

**Comment by ptitSeb** [comment]
Yes, glMapBuffer never return a real VBO, because OpenGL VBO can be read back were GLES cannot.
Real VBO are still used under the hood.

Not sure what the issue is, but that point (glMapBuffer not returning a real VBO) is normal. I have tried many games that used VBO, and all seems to works now. But there is maybe a bug.

**Comment by RusJJ** [comment]
Crashing after that commit: https://github.com/ptitSeb/gl4es/commit/0b173f318dd8a0a4232300cb5264487fb7d17415

**Comment by ptitSeb** [comment]
crashing were, how. Can you be more specific? I don't have crash on my side, everything works fine, so it's difficult to fix anything with no info.

**Comment by RusJJ** [comment]
This is gl4es. Renamed.
![IMG_20201107_234608](https://user-images.githubusercontent.com/8864329/98451099-c2416580-2153-11eb-811f-c4d30d78c894.jpg)

**Comment by ptitSeb** [comment]
Yeah, I recognise function names.

Can you build with "-g" so the backtrace have lines numbers? That would help a bit, next time.

From what I understand, the bind is not done before the drawing, on a "glDrawRangeElements". I'll try to see how this can go wrong in this case.
